designs/tripleo/osc-tripleo.md

```diff
@@ -0,0 +1,137 @@
+# OSC TripleO
```

Comment:
what is the plan to add these templates? meaning which repository. I see ODL as separate repository for puppet etc in ODL github?

Reply:
Do you plan to add a note how to work with TripleO community, to get approval for these templates?

Reply:
The document mentions that here "The goal is for all the changes described on this section to be upstreamed to the [THT repository](#tripleo-repo)"  and "### [TripleO Repo](https://github.com/openstack/tripleo-heat-templates) . ODL was just an example we were looking at. ODL has changes on the same repo and also on the puppet-tripleo repo on GitHub. I am assuming we wont need the latter, just the repo https://github.com/openstack/tripleo-heat-templates. "How" we will work with the tripleo community is outside of design IMO we will definitely need to describe that through RFS (we already have), blueprints, etc. But the process is orthogonal to the design and needs to be discussed more with RH.